src/harness/testcases/WINNF_FT_S_FPR_testcase.py

```diff
@@ -213,7 +214,7 @@ def generate_FPR_1_default_config(self, filename):
     dump_records_iteration_0_sas_test_harness_1 = {
         'cbsdRecords': cbsd_fad_records_iteration_0_sas_test_harness_1
     }
- 
+
 
     # Create the actual config.
     iteration0_config = {
```

Comment:
line: 225  Please   update  to “'clientCert': sas.GetDefaultDomainProxySSLCertPath()”
line: 226  Please   update  to “ 'clientKey':   sas.GetDefaultDomainProxySSLKeyPath() “

Reply:
This was resolved with the merge from MCP.1-fix.